08-eval-platform/requirements-dev.txt

```diff
@@ -2,7 +2,3 @@ pytest
 python-dotenv[cli]
 pytest-retry
 pytest-vcr
-arize-phoenix
```

Comment:
phoenix and evals are used in the eval_job, which isn't a test. other deps weren't used anymore